alma/controllers/hook/DisplayPaymentHookController.php

```diff
@@ -70,7 +70,7 @@ public function run($params)
             Tools::ps_round((float) $this->context->cart->getOrderTotal(true, Cart::BOTH), 2)
         );
 
-        foreach ($installmentPlans as $plan) {
+        foreach ($installmentPlans as $keyPlan => $plan) {
             $installment = $plan->installmentsCount;
             $key = "general_{$installment}_{$plan->deferredDays}_{$plan->deferredMonths}";
```

Comment:
Do you have a "Utils" method to generate the key  always in the same way ?

Reply:
@Francois-Gomis 
I don't think, but the key is just for the id in html and js code to make an uniq identity. What do you mean by the same way ?

Reply:
You use the same "method" in paymentOptionsHookController .

Reply:
@Francois-Gomis 
I created a new method `keyForInstallmentPlan()` because `keyForFeePlan()` didn't match for this feePlan.